tests/client.rs

```diff
@@ -689,7 +729,9 @@ fn staking() {
     .unwrap_err();
     // FIXME currently this results in an unexpected failure because of how snarkvm handles integer overflow errors
     // this should be improved to properly handle execution errors internally and showing a clear error message in the CLI
-    assert!(error.contains("Integer subtraction failed"));
+    assert!(
+        error.contains("Integer subtraction failed") || error.contains("Subtraction underflow")
```

Comment:
This error difference should be behind the compiler flag so that it's more self-explanatory.